metals/src/main/scala/scala/meta/internal/decorations/SyntheticsDecorationProvider.scala

```diff
@@ -559,6 +559,19 @@ final class SyntheticsDecorationProvider(
         case m: MethodSignature =>
           val decoration = ": " + decorationPrinter.printType(m.returnType)
           Some(decorationOptions(lspRange, decoration))
+        // local type aliases
+        case ValueSignature(tp: s.TypeRef) if tp.symbol.isLocal =>
+          textDocument.symbols.collectFirst {
```

Comment:
what if type alias is defined in other file? It isn't local then.

Reply:
Then it should work normally, the problem is with printing local type aliases as far as I was able to check. Or do you have a counter example?

Reply:
Changed the test to include a toplevel object.

Reply:
Right, non local type was working, my bad.